presto-hudi/pom.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.facebook.presto</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>0.231-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-hudi</artifactId>
+    <description>Presto - Hudi Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-hive-hadoop2</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-hive</artifactId>
+        </dependency>
+
+        <dependency>
```

Comment:
When I suggested to extract a separate module - I thought that Hudi is something that is not open source.

Since it is opensource, is there's a good reason why don't we want to just have it included directly in `presto-hive`  and entirely avoid unnecessary interfacing?

CC: @wenleix @shixuan-fan

Reply:
Hudi is opensourced and is indeed supported by Google Cloud and AWS EMR :) 

Regarding your comment on moving this back to presto-hive, we could still continue with presto-hudi as a connector module making it a placeholder for longterm. Any further feature requests from Hudi on Presto side can levergae this.  But please let me know your thoughts.

Reply:
From what I understand, at the end of the day Hudi is just yet another table format. It is quite similar to ORC / Parquet, but apparently a little bit broader scoped as it also specifies the way files have to be organized.

Since Parquet / ORC are first class citizens in `presto-hive` i wonder why Hudi cannot be so?
Eventually we may want to introduce Hudi as a first class citizen allowing table writes in this format. We may want to add it as an option to our list of officially supported formats specified by the https://github.com/prestodb/presto/blob/master/presto-hive-metastore/src/main/java/com/facebook/presto/hive/HiveStorageFormat.java#L42

@wenleix @shixuan-fan @highker Do you have any objections against including Hudi into presto-hive module?

Reply:
Since we just worked on another connector, and have a pending PR on a connector for Alluxio metadata service (#13986), you may find some of our experience useful. 

We initially started down the path of having our own connector, similar to how you have `presto-hudi`, it gives us a lot more flexibility, and allows us to use the additional statistics to affect query plan, but the cost of that is we had to write more code to get a working version. We switched to just implementing a meta-store in `presto-hive-metastore`, it was a much simpler path, but we lose some of that flexibility. In your case, you may find merging into `presto-hive` restricting as well, because hudi is more than just a file type.

Reply:
Thanks @haoyuan for pointing this. I ll take a look at this.